packs/data/kingmaker-bestiary.db/imeckus-stroon.json

```diff
@@ -0,0 +1,3714 @@
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [],
+                "slug": null,
+                "source": {
+                    "value": ""
+                },
+                "traits": {
+                    "custom": "",
+                    "rarity": "common",
+                    "value": [
+                        "concentrate",
+                        "metamagic"
+                    ]
+                },
+                "trigger": {
+                    "value": ""
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "xtGHTPAuSTsudXMG",
+            "img": "systems/pf2e/icons/actions/Passive.webp",
+            "name": "Enhanced Familiar",
+            "sort": 3300000,
+            "system": {
+                "actionCategory": {
+                    "value": ""
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p>Imeckus Stroon's familiar Lintwerth is a loyal companion and is a bit more powerful than the standard familiar in that he does not possess the minion trait; Imeckus can direct Lintwerth's actions verbally as a free action. If Lintwerth is slain, Imeckus becomes @UUID[Compendium.pf2e.conditionitems.Drained]{Drained 1} for 24 hours.</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [],
+                "slug": null,
+                "source": {
+                    "value": ""
+                },
+                "traits": {
+                    "custom": "",
+                    "rarity": "common",
+                    "value": []
+                },
+                "trigger": {
+                    "value": ""
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "moA510VYoGxOfach",
+            "img": "systems/pf2e/icons/actions/OneAction.webp",
+            "name": "Recall Lintwerth",
+            "sort": 3400000,
+            "system": {
+                "actionCategory": {
+                    "value": ""
+                },
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 1
+                },
+                "description": {
+                    "value": "<p>Imeckus calls out Lintwerth's name and snaps his fingers, causing his dust mephit familiar (and up to 1 Bulk of objects carried by Lintwerth) to instantly transport to an adjacent square of Imeckus' choice, providing Lintwerth is within a mile.</p>"
```

Comment:
Do we want to link dust mephit? Or I guess it's a familiar and NPC familiars are usually even weirder than PC ones.

Reply:
Lintwerth is an NPC entry in its own right. We shouldn't link to the base Dust Mephit